src/edges_estimate/foregrounds.py

```diff
@@ -302,3 +302,44 @@ def calculate(self, ctx, **params):
                     break  # only add to one thing, otherwise it's doubling up.
                 else:
                     ctx[key] *= bias
+
+
+@attr.s
+class LogPoly(Foreground):
```

Comment:
@PSims It would be nice to get your review on this :)

Reply:
This looks reasonable to me! 
As an additional check I would suggest building a mock data set constructed to follow this parametrisation and then make sure that you recover the result you expect when fitting it.